scripts/fuzz_opt.py

```diff
@@ -1405,6 +1405,20 @@ def wasm_has_duplicate_tags(wasm):
     return binary.count(b'jstag') >= 2 or binary.count(b'wasmtag') >= 2
 
 
+# Detect whether there is a trap reported before an export call in the output.
+def traps_in_instantiation(output):
+    trap_index = output.find(TRAP_PREFIX)
+    if trap_index == -1:
+        # In "fixed" output, traps are replaced with *exception*.
+        trap_index = output.find('*exception*')
+        if trap_index == -1:
+            return False
+    call_index = output.find(FUZZ_EXEC_CALL_PREFIX)
+    if call_index == -1:
+        return True
+    return trap_index < call_index
```

Comment:
There is `INSTANTIATE_ERROR` which I think may be simpler to use?

Reply:
Nice, I'll try that out.

Reply:
Hmm, `scripts/fuzz_shell.js` does as well, but yeah, good point, `wasm-opt` doesn't.